ibm/data_source_ibm_scc_account_location.go

```diff
@@ -0,0 +1,144 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/scc-go-sdk/adminserviceapiv1"
+)
+
+func dataSourceIbmSccAccountLocation() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIbmSccAccountLocationRead,
+
+		Schema: map[string]*schema.Schema{
+			"location_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The programatic ID of the location that you want to work in.",
+			},
+			"id": &schema.Schema{
```

Comment:
Don't need a seperate "id" attribute to be defined in scheam since Terraform add by default for resoruces and datasources 
d.SetID(XXXX)